tests/login.ts

```diff
@@ -0,0 +1,14 @@
+import { Page } from "@playwright/test";
+
+export const login = async (
+  page: Page,
+  username: string = "admin_user",
+  password: string = "admin123",
+) => {
+  await page.goto("/");
+  await page.getByLabel("Username").click();
+  await page.getByLabel("Username").fill(username);
+  await page.getByLabel("Username").press("Tab");
```

Comment:
Could probably use `const usernameField = page.getByLabel("Username")` and reuse, not a big deal though.